README.md

```diff
@@ -107,6 +107,9 @@ Also there is an option to functionally test the operator on an existing cluster
     - PROTECTED_SUBNET_TEST1
     - PROTECTED_SUBNET_TEST2 (list of protected subnets to test, if either of them are empty then no protected subnet test will run)
 
+### Handling Restricted Pod Security Admission (PSA)
+The fvt test script attempts to install `static-route-operator` and `hostnet` pods into the `default` namespace. These pods require escalated privileges to be able to verify functionality of the operator. Since [Kubernetes 1.25](https://kubernetes.io/docs/concepts/security/pod-security-admission/), PSA has been a stable feature and allows users to restrict or prevent the creation of pods that require higher levels of authority. The fvt test script will temporarily apply labels to the `default` namespace to allow the privileged pods to be created, and upon completion of the fvt test script the label values will be removed (if they did not exist prior to testing) or they will be reset to their previous values (if they were set prior to running the test script). 
```

Comment:
Should those pods be using a diff namespace other than `default` ?
Does the `static-route-operator` logic need to get modified for PBA ?

Reply:
The open source `static-route-operator` installs all resources into the `default` namespace. The IBM managed `static-route-operator` add-on installs everything into the `kube-system` namespace.

The IBM `static-route-operator` add-on will not run into these PSA issues since it is installed into the `kube-system` namespace.

As for moving the pods into a namespace other than `default`, it was decided yesterday to leave it as-is for now. The responsibility will be on the user to move it to a different namespace if they require it. This update is just to ensure that the FVT tests can still be run to ensure functionality.